### DISCUSSION TITLE
Cannot invoke a function from Python

### DISCUSSION THREAD

**Discussion by AnandR-Techtinium** [supabase/supabase#14014] [Questions]
Trying to call a supabase function from Python using https://supabase.com/docs/reference/python/functions-invoke.

But I am getting {'data': b'Function not found', 'error': None}

Here is my code:

```
func = supabase_client.functions()
async def test_func(loop):
        resp = await func.invoke('helloworld1', invoke_options={'body':{}})
        return resp
loop = asyncio.get_event_loop()
resp = loop.run_until_complete(test_func(loop))
loop.close()
```

**Comment by TotallyIntended** [comment]
I have the same issue. 

A current workaround for me is using the .rpc() function of the client to execute a function. 
However, due to an apparent incompatibility between the Supabase API and the postgrest python module (see  [here](https://github.com/supabase-community/supabase-py/issues/405)), you cannot receive return values. 

A (stupid) workaround for that is to save the return value in a table and retrieve it through a select query: 


```
client.rpc("your_function_name", params={[YOUR PARAMS]}).execute()
resp = client.table("your_table_name").select("*").order("id_desc").limit(1).execute()
```

**Comment by CareTiger** [comment]
this issue still pending?  i just get a 

`{'data': b'', 'error': None}`

here is my code

func = supabaseClient.functions()

```
async def test_func():
    print("test_func")
    resp = await func.invoke("/user/profile",
                             invoke_options={
                                    'method': 'GET'
                             })
    return resp

task = asyncio.create_task(test_func());
await task
```
